scripts/explorer-uss-start.sh

```diff
@@ -39,4 +39,7 @@ _BPX_JOBNAME=${JOB_NAME} $NODE_BIN $SERVER_DIR/src/index.js \
 	--key  $KEYSTORE_KEY \
 	--cert $KEYSTORE_CERTIFICATE \
 	--csp ${ZOWE_EXPLORER_FRAME_ANCESTORS} \
+	--keyring $KEYRING_NAME \
+	--keyring-owner $KEYRING_OWNER \
+	--keyring-label $KEY_ALIAS \
```

Comment:
if we pass both 
--key and --cert takes precedence over 
and keyring params - keyring owner and label

we need instance environment conditional to pass one set of params either key-cert or keyring

Reply:
if [$ZOWE_KEYRING] - pass keyring params
else pass regular keystore paths

Reply:
other options is 
if KEYSTORE_KEY,  KEYSTORE_CERTIFICATE  are both blank [''] they it may pick keyring params

Reply:
try locally with explorer-ui-server right set of param:
npm start -- -s "uss" -b "/" -d "public" -p "9090" --key '' --cert '' -n "keyring" -o "keyr 
ing-owner" -l "keyring-label"

- it try to load keyring when --key & --cert are blank

Reply:
@NakulManchanda  it was working with the second option but I didn't notice that your refactor work changed that. If $KEYSTORE_KEY and $KEYSTORE_CERTIFICATE will be empty, the command will look like this:
`npm start -- -s "uss" -b "/" -d "public" -p "9090" --key --cert  -n "keyring" -o "keyring-owner" -l "keyring-label"`
i can make change to explorer-ui-server to ignore boolean values of _key_ and _cert_, but if you put `--key ''` it will be evaluated as a string containing quotes. So it not work this way. Will this change be ok?

Reply:
PR for proposed change 
https://github.com/zowe/explorer-ui-server/pull/33